File: app/src/main/java/com/gnommostudios/alertru/alertru_android/view/fragments/InformationFragment.java
package com.gnommostudios.alertru.alertru_android.view.fragments;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ScrollView;

import com.gnommostudios.alertru.alertru_android.R;

public class InformationFragment extends Fragment {

    private ScrollView scrollView;

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        scrollView = (ScrollView) getActivity().findViewById(R.id.scrollViewInformation);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_information, container, false);
    }

}

File: app/src/main/java/com/gnommostudios/alertru/alertru_android/view/fragments/InformationFragment.kt
package com.gnommostudios.alertru.alertru_android.view.fragments

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ScrollView

import com.gnommostudios.alertru.alertru_android.R

class InformationFragment : Fragment() {

    private var scrollView: ScrollView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        scrollView = activity.findViewById<View>(R.id.scrollViewInformation) as ScrollView
    }

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        // Inflate the layout for this fragment
        return inflater!!.inflate(R.layout.fragment_information, container, false)
    }

}
